.setup/vagrant/setup_vagrant.sh

```diff
@@ -14,6 +14,16 @@ if [ -x "$(command -v pip3)" ]; then
     python3 ${SUBMITTY_REPOSITORY}/.setup/bin/reset_system.py
 fi
 
+OPTSTRING=":c"
+while getopts ${OPTSTRING} opt; do
+    case ${opt} in
+    c)
+    echo "This file is used to check if Submitty is being run in the Github Actions CI." > ${SUBMITTY_REPOSITORY}/.github_actions_ci_flag
```

Comment:
Writing a file for one-time use like this feels like a hack.  Is there a way to use an environment variable instead?

Reply:
Talked to Barb in the meeting yesterday and she said that since vagrant and utm are doing this, she liked this way. We can discuss more in the meetings